app/src/main/java/com/example/agreementSample/MainActivity.kt

```diff
@@ -1,11 +1,45 @@
 package com.example.agreementSample
 
-import androidx.appcompat.app.AppCompatActivity
+import android.content.pm.ActivityInfo
 import android.os.Bundle
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import androidx.databinding.DataBindingUtil
+import com.example.agreementSample.databinding.ActivityMainBinding
 
 class MainActivity : AppCompatActivity() {
+
+    private var mBinding: ActivityMainBinding? = null
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main)
+        requestedOrientation =  ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE
+        mBinding = DataBindingUtil.setContentView(this, R.layout.activity_main)
+        setup()
+    }
+
+    private fun setup() {
+        setForwardText("Avançar")
```

Comment:
Use localized strings

Reply:
That's the sample module. It's just to show/test the use of it.